Source/WTF/wtf/text/icu/UTextProviderLatin1.cpp

```diff
@@ -301,7 +306,7 @@ static void textLatin1ContextAwareMoveInPrimaryContext(UText* text, int64_t nati
     text->nativeIndexingLimit = text->chunkLength;
     text->chunkOffset = forward ? 0 : text->chunkLength;
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-    StringImpl::copyCharacters(const_cast<UChar*>(text->chunkContents), unsafeMakeSpan(static_cast<const LChar*>(text->p) + (text->chunkNativeStart - text->b), text->chunkLength));
+    StringImpl::copyCharacters(chunkSpan(text), unsafeMakeSpan(static_cast<const LChar*>(text->p) + (text->chunkNativeStart - text->b), text->chunkLength));
```

Comment:
This code can’t be right! I can’t find any code anywhere that is setting `text->p`.

Reply:
I left it as is for now as I'm not quite sure what to do about it. Via code auditing, it doesn't obviously look unused. there is code calling into it as far as I can see. However, when I run the layout tests, I don't see this code path getting exercised indeed..